src/wait-for-element-to-be-removed.js

```diff
@@ -0,0 +1,64 @@
+import {getDocument, getSetImmediate, newMutationObserver} from './helpers'
+
+function waitForElementToBeRemoved(
+  callback,
+  {
+    container = getDocument(),
+    timeout = 4500,
+    mutationObserverOptions = {
+      subtree: true,
+      childList: true,
+      attributes: true,
+      characterData: true,
+    },
+  } = {},
+) {
+  return new Promise((resolve, reject) => {
+    if (typeof callback !== 'function') {
+      reject(
+        'waitForElementToBeRemoved requires a function as the first parameter',
```

Comment:
I think this should be an error object rather than just a string so we get a stacktrace.